test/models/validator_test.rb

```diff
@@ -183,11 +181,11 @@ class ValidatorTest < ActiveSupport::TestCase
     assert result.last.delinquent?
   end
 
-  test "#filtered_by inactive excludes correct validators from collection" do
-    result = Validator.filtered_by(["inactive"])
+  test "#filtered_by active excludes correct validators from collection" do
```

Comment:
Please fix this test name. Filtering by active (or anything else), doesn't exclude any validators. On the contrary, it includes all validators that meet condition (in this case - validators that are active), but they also may or may not meet other conditions (eg. might be also active and private) - it doesn't matter, as long as they meet selected filter. While I understand what test author wanted to say, this test description is confusing and incorrect.
It should be something like `#filtered_by active includes all active validators in collection`